cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -913,6 +914,18 @@ function Configure-CniNetworking {
   Log-Output "CNI config:`n$(Get-Content -Raw ${l2bridge_conf})"
 }
 
+# Obtain the host dns conf and save it to a file so that kubelet/CNI can use it to configure
+# dns suffix search list for pods
+function Configure-HostDNSConf {
+  $server_ip = Get-DnsClientServerAddress -InterfaceAlias "vEthernet (Ethernet)" | Select-Object -ExpandProperty ServerAddresses
```

Comment:
The "vEthernet (Ethernet)" name could change in future Windows versions. We have `Get_MgmtNetAdapter` already in this file to help cope if it does change - can you just call `Get_MgmtNetAdapter` here and then pass the adapter it returns to `Get-DnsClientServerAddress`?

Reply:
Updated.